content/dstu2/authorization.md

```diff
@@ -68,11 +68,21 @@ resources and exchange context information with client applications.
 
 ## Registration ##
 In order for your client application to utilize any protected resources, 
-your client application must first register using our 
-[code console][CERNER-CODE-CONSOLE]. A Cerner Care account is required to 
+your client application must first register. A Cerner Care account is required to 
 register applications. Obtaining a Cerner Care account is free and 
 requires nothing but an email address.
 
+If registering an application that will not be using offline_access, and that is not 
```

Comment:
Can this be stated more positively, mentioning a provider or patient access workflow with a logged-in user?